lxd/device/nic_ovn.go

```diff
@@ -747,6 +822,7 @@ func (d *nicOVN) postStop() error {
 			"last_state.hwaddr":        "",
 			"last_state.mtu":           "",
 			"last_state.created":       "",
+			"last_state.vdpa.name":     "",
```

Comment:
We need this added to the docs in doc/reference/instance_options.md, search for "last_state.vf.id" for an example.

Reply:
sure